src/layouts/HeaderLayout.tsx

```diff
@@ -18,11 +18,6 @@ const HeaderLayout: React.FC<HeaderLayoutProps> = ({ children, title }) => {
   const headName = `${title} | CourtCanva`;
   return (
     <motion.article
```

Comment:
Yes, this is the correct place for removing the transition of our pages but do you think we still need an empty motion.article tag?